internal/hcp/registry/types.bucket.go

```diff
@@ -68,13 +68,13 @@ func (b *Bucket) ReadFromHCLBuildBlock(hcpBlock *hcl2template.BuildBlock) {
 	if b == nil {
 		return
 	}
-	b.Description = hcpBlock.Description
```

Comment:
To avoid confusion, we should probably rename `hcpBlock` to `buildBlock` also, what do you think?

Reply:
Yeah good call.